metal3-scaling-experiments/delete-clusters.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
I won't repeat the same notes as from above script, they seem to all apply here as well.